pkg/monitor/datapath_sock_trace.go

```diff
@@ -0,0 +1,32 @@
+package monitor
+
+import "github.com/cilium/cilium/pkg/types"
+
+// Socket trace event point with respect to service translation
+const (
+	XlatePointPreDirectionFwd = iota
```

Comment:
This is now off by one:

```suggestion
	XlatePointUnknown = iota
	XlatePointPreDirectionFwd
```